v7/nanogallery_directive/templates/mako/embedded-nanogallery.tmpl

```diff
@@ -35,8 +35,8 @@
   // We need window.onload here because jQuery is not loaded yet
   window.onload = function() {
       $("#nanoGallery3").nanoGallery({
-      % for k, v in options.iteritems():
-        % if isinstance(v, unicode):
+      % for k, v in options.items():
+        % if isinstance(v, str):
```

Comment:
Is it accessible from within templates?  (and why is there so much logic in this template?)